packages/frontend/core/src/pages/workspace/all-page/all-page.tsx

```diff
@@ -25,6 +34,19 @@ export const AllPage = () => {
     filters: filters,
   });
 
+  const isActiveView = useIsActiveView();
+
+  useEffect(() => {
+    if (isActiveView) {
+      globalContext.isAllDocs.set(true);
+
+      return () => {
+        globalContext.isAllDocs.set(false);
+      };
+    }
```

Comment:
may abstract this pattern into a new hook

Reply:
I'll think about how to abstract